src/layer/marker/Icon.Default.js

```diff
@@ -20,41 +22,14 @@ import * as DomUtil from '../../dom/DomUtil';
 export var IconDefault = Icon.extend({
 
 	options: {
-		iconUrl:       'marker-icon.png',
-		iconRetinaUrl: 'marker-icon-2x.png',
-		shadowUrl:     'marker-shadow.png',
+		iconUrl:     emptyImageUrl,
+		shadowUrl:   emptyImageUrl,
 		iconSize:    [25, 41],
 		iconAnchor:  [12, 41],
 		popupAnchor: [1, -34],
 		tooltipAnchor: [16, -28],
-		shadowSize:  [41, 41]
-	},
-
-	_getIconUrl: function (name) {
-		if (!IconDefault.imagePath) {	// Deprecated, backwards-compatibility only
-			IconDefault.imagePath = this._detectIconPath();
```

Comment:
We should not remove that, otherwise we lose ability to specify just icon filenames.
This would break everything.

Reply:
I guess I could add this back, and put in some logic so that if `L.Icon.Default.imagePath` is specified, the CSS class is removed. Adding a `className` option was the quickest to implement, but it would be possible to a it conditionally before the code from `L.Icon` needs it.